theano/tensor/subtensor.py

```diff
@@ -2304,7 +2304,10 @@ def R_op(self, inputs, eval_points):
 
 def take(a, indices, axis=None, mode='raise'):
     a = theano.tensor.as_tensor_variable(a)
-    indices = theano.tensor.as_tensor_variable(indices)
+    if(indices == []):
```

Comment:
What happen with an empty tuple? Can you test that? If numpy behave on an empty tuple like an empty list, make sure to do the same.

Also add test.